src/tests/networking_tests.rs

```diff
@@ -2,6 +2,7 @@
 pub mod net_tests {
     use super::super::*;
     use crate::interface;
+    use crate::interface::errnos::{syscall_error, Errno};
```

Comment:
why did you need to import this?

Reply:
Wasn't necessary, i figured out how to use errno